ansible_base/rbac/triggers.py

```diff
@@ -131,7 +131,7 @@ def rbac_post_init_set_original_parent(sender, instance, **kwargs):
     instance.__rbac_original_parent_id = getattr(instance, parent_id_name)
 
 
-def get_parent_ids(instance) -> list[tuple[int, int]]:
+def get_parent_ids(instance) -> list[tuple[Model, int]]:
```

Comment:
In general, should we ever try and get more specific than `Model`? i.e. if its guaranteed to be a specific model should that be added to the type hint?

Reply:
I'm going to say "no" because these are models from the registry, and the app using this libraries puts its own models in there. The criteria for what goes into the registry fits `Model` like a glove. It is almost a perfect description of the expected types.